### PR TITLE
Feat/add to backlog

### LINKED ISSUE #64: Add game with metadata to backlog

### PR DESCRIPTION
**Pull request by hola-soy-milk**
Closes #64 

image
image

### CODE REVIEW

File: renderer/components/Rom/index.tsx [hunk 1]
```diff
@@ -39,6 +41,22 @@ export const Rom = ({
...
     <button onContextMenu={handleContextMenu} className="w-full">
       <Menu id={MENU_ID}>
+        {rom?.info && (
+          <Item
+            id="add-to-backlog"
+            onClick={async () => {
+              const res = await fetch(`/api/hltb?game=${rom.info?.title}`);
+              const items: HLTGame[] = await res.json();
+
+              if (items.length > 0) {
```
Comment (SaraVieira) [inline renderer/components/Rom/index.tsx:52]: Hmm, I'm thinking we should still add it if it doesn't find a match, we just don't have the hours

File: renderer/components/Rom/index.tsx [hunk 2]
```diff
@@ -39,6 +41,22 @@ export const Rom = ({
...
+                await addToBacklog(items[0]);
+                router.push("/backlog");
```
Comment (SaraVieira) [inline renderer/components/Rom/index.tsx]: That sounds intrusive for the user no?

File: renderer/pages/[folder]/[path]/[file]/index.tsx [hunk 3]
```diff
@@ -56,6 +62,16 @@ export const Files = () => {
     setEditing(false);
   };
 
+  const add = async () => {
+    const res = await fetch(`/api/hltb?game=${activeRom.info?.title}`);
+    const items: HLTGame[] = await res.json();
+
+    if (items.length > 0) {
+      await addToBacklog(items[0]);
+      router.push("/backlog");
```
Comment (SaraVieira) [inline renderer/pages/[folder]/[path]/[file]/index.tsx]: Same comments

Comment (SaraVieira) [reply renderer/pages/[folder]/[path]/[file]/index.tsx]: Also maybe add a toast saying game.name was added

File: renderer/components/Rom/index.tsx (hunk 1 above)
Comment (hola-soy-milk) [reply renderer/components/Rom/index.tsx:52]: Ahhh yes this makes much more sense, ty!

File: renderer/pages/[folder]/[path]/[file]/index.tsx (hunk 3 above)
Comment (hola-soy-milk) [reply renderer/pages/[folder]/[path]/[file]/index.tsx]: Ah! Is the intrusivity the navigating to the backlog?

We could replace that with the toast, what do you think? The toast is a great idea imo

Comment (SaraVieira) [reply renderer/pages/[folder]/[path]/[file]/index.tsx]: Yes sounds good!

Comment (hola-soy-milk) [reply renderer/pages/[folder]/[path]/[file]/index.tsx]: Amazing thanks for confirming!!

**Comment by hola-soy-milk** [comment]
Ok I think I've got a good solution for both the concerns now! wdyt? :)

**Review by SaraVieira — APPROVED** [review]

**Pull request merged**